EventListener/SentryExceptionListener.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MauticPlugin\SwisSentryBundle\EventListener;
+
+use Symfony\Component\HttpKernel\Event\ExceptionEvent;
+use Symfony\Component\HttpKernel\Event\TerminateEvent;
+
+final class SentryExceptionListener
+{
+    public function __construct()
+    {
+        if (!getenv('SENTRY_DSN')) {
+            return;
+        }
+
+        \Sentry\init([
+            'dsn' => getenv('SENTRY_DSN'),
+            'environment' => getenv('APP_ENV') ?: 'unknown',
```

Comment:
Please add `SENTRY_ENVIRONMENT` as a value,:

```suggestion
            'environment' => getenv('SENTRY_ENVIRONMENT') ?: getenv('APP_ENV') ?: 'unknown',
```